include/clover_field_order.h

```diff
@@ -609,10 +585,7 @@ namespace quda {
 	 */
 	__device__ __host__ inline void load(real v[block], int x, int parity, int chirality) const
         {
-          norm_type nrm;
-          if (isFixed<Float>::value) {
-            nrm = vector_load<float>(norm, parity * norm_offset + chirality * stride + x);
-          }
+          norm_type nrm = isFixed<Float>::value ? vector_load<float>(norm, parity * norm_offset + chirality * stride + x) : 0;
```

Comment:
I know the value of `nrm` is numerically irrelevant in the non-fixed case since it isn't used in the non-fixed flavor of `copy_and_scale`, but I think it'd be more "aesthetically" pleasing if we used `1` instead of `0` here. otoh, if `0` is useful for compiler optimizations (for whatever reason), then I'm fine with keeping it as is.

Reply:
Switching to 1.0 seems reasonable (fd5ff4376)